pkg/common/cns-lib/volume/manager.go

```diff
@@ -54,19 +54,34 @@ const (
 	taskInvocationStatusInProgress = cnsvolumeoperationrequest.TaskInvocationStatusInProgress
 	taskInvocationStatusSuccess    = cnsvolumeoperationrequest.TaskInvocationStatusSuccess
 	taskInvocationStatusError      = cnsvolumeoperationrequest.TaskInvocationStatusError
+
+	// CSITaskInfoEmptyFault is the fault type when taskInfo is empty.
+	CSITaskInfoEmptyFault = "csi.fault.TaskInfoEmpty"
```

Comment:
I don't think the error strings should be defined here. They are prefixed with CSI & the package is CNS-lib. IMO, they don't belong to this package